public/javascripts/index.js

```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+ReactDOM.render(<a href="#">Capital A</a>, document.getElementById('buttons'));
+
+ReactDOM.render(<h1>A</h1>, document.getElementById('letters'));
```

Comment:
Yeah that's a good catch. I've already changed it because this doesn't seem to work when the rendering involves many React classes